src/components/Icon/index.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'underscore';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import themeColors from '../../styles/themes/default';
+import variables from '../../styles/variables';
+import * as BrandAssets from './BrandAssets';
+import * as Expensicons from './Expensicons';
+
+const ICONS = _.extend(BrandAssets, Expensicons);
+
+const propTypes = {
+    /* The asset to render. */
+    icon: PropTypes.oneOf(_.values(ICONS)).isRequired,
```

Comment:
`<Icon icon="" />` is repetitive, so I would suggest changing this prop name to `src`.